Update recorder download location (#2957)
elastic/observability-docs: docs/en/observability/synthetics-recorder.asciidoc

[[synthetics-recorder-set-up]]
== Set up

> Details of where to download the Elastic Synthetics Recorder from can be found on the https://github.com/elastic/synthetics-recorder/blob/main/docs/DOWNLOAD.md[download page].

Reviewer: ```suggestion
For information on how to download the Elastic Synthetics Recorder, go to the https://github.com/elastic/synthetics-recorder/blob/main/docs/DOWNLOAD.md[download page].
```
